src/components/auth/SettingsModal.tsx

```diff
@@ -41,7 +41,7 @@ const missingMoxfieldProfileImage = "https://upload.wikimedia.org/wikipedia/comm
 
 export const SettingsMenuItem = React.memo(function SettingsMenuItem({ finalRef }: { finalRef: any }) {
     const { isOpen, onOpen, onClose } = useDisclosure();
-    const setFavoriteCommander = ProfileService.useSetFavoriteCommander();
+    const setServerProfile = ProfileService.useSetServerProfile();
```

Comment:
nit: call this setProfile

Reply:
changed to updateProfile, as requested in later comment